### PR TITLE
Add Commons POTY 2021

### PR DESCRIPTION
**Pull request by rwjuk**
This PR adds the eligibility check for POTY 2021 based on the template for past events.

Thanks!

### CODE REVIEW

File: tool-labs/accounteligibility/framework/EventFactory.php [hunk 1]
```diff
@@ -14,14 +14,22 @@ class EventFactory
      */
     public function getDefaultEventID()
     {
-        return 56;
+        return 57;
```
Comment (zhuyifei1999) [inline tool-labs/accounteligibility/framework/EventFactory.php:17]: I think this should be 58

File: tool-labs/accounteligibility/framework/EventFactory.php [hunk 2]
```diff
@@ -14,14 +14,22 @@ class EventFactory
...
 
     /**
      * Get all available events.
      */
     public function getEvents()
     {
+        ##########
+        ## 2021: Commons Picture of the Year for 2020
+        ##########
+        yield (new Event(57, 2021, 'Commons Picture of the Year for 2020', 'https://commons.wikimedia.org/wiki/Commons:Picture_of_the_Year/2020'))
```
Comment (zhuyifei1999) [inline tool-labs/accounteligibility/framework/EventFactory.php:28]: 57 is duplicate

File: tool-labs/accounteligibility/framework/EventFactory.php (hunk 1 above)
Comment (Pathoschild) [reply tool-labs/accounteligibility/framework/EventFactory.php:17]: I'll fix these after merging. Thanks for pointing them out!

Comment (Pathoschild) [reply tool-labs/accounteligibility/framework/EventFactory.php]: Fixed. Thanks for pointing these out!

File: tool-labs/accounteligibility/framework/EventFactory.php (hunk 2 above)
Comment (Pathoschild) [reply tool-labs/accounteligibility/framework/EventFactory.php]: Fixed!

**Comment by Pathoschild** [comment]
Merged and deployed. Thanks for the pull request!

**Pull request merged**